### DISCUSSION TITLE
My Account Blocking

### DISCUSSION THREAD

**Discussion by lihuhuhu** [community/community#119691] [Other Feature Feedback, Questions, & Ideas]
My account "github123666" has been banned and I haven't received any emails about it. It is possible that I was hacked and violated the community policy, and thus my account was banned. I hope you can give me a chance, this account is very important to me, there is very important code in it, I hope you can give me an answer.

**Comment by Kuntal-Gain** [comment]
Same for mine in yesterday, Support keep ignoring  they haven't send any reason mail why suspending account

**Comment by ghostinhershell** [comment]
Hi @lihuhuhu, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them.
